ArjunaCore/arjuna/pom.xml

```diff
@@ -171,11 +171,6 @@
           </excludes>
         </configuration>
       </plugin>
-      <plugin>
```

Comment:
I am not sure if it's right or wrong :) But I will say that maven-install-plugin and I think the deployment part of our build choices was complicated and an aim was to only deploy the main artifacts and their truly needed dependencies (in other words the uber jars and their parent poms and presumably the bom).

I don't know if this pull request changes this (or indeed if that is the current status) but I remember somehow spending some time try to make sure only certain dependencies were deployed to Maven at some point and so wanted to mention as I do remember it being fiddly and so might account for the lack or normalisation already?